doc/manuals/advancedconfig.rst

```diff
@@ -0,0 +1,97 @@
+directory first.  Each of the tools in the *Compute* menu loads a configuration file when it is run.  These have names starting with a "gui\_” prefix.  For example:
+
+
+.. table:: 
+   :align: center
+
+   +---------------------------+----------------------------------+
+   | Compute Menu Name         | Configuration File Entry Point   |
+   +===========================+==================================+
+   | Track Features            | gui_track_features.conf          |
+   +---------------------------+----------------------------------+
+   | Estimate Camera/Landmarks | gui_initialize.conf              |
+   +---------------------------+----------------------------------+
+   | Save Frames               | gui_frame_image_writer.conf      |
+   +---------------------------+----------------------------------+
+   | Batch Compute Depth Maps  | gui_compute_depth.conf           |
+   +---------------------------+----------------------------------+
+   | Fuse Depth Maps           | gui_integrate_depth_maps.conf    |
+   +---------------------------+----------------------------------+
+
+
+Most of these configuration files also reference other configuration files with an “include” statement.  Configuration values for tools can also be added to the project file but 
+copying the GUI configuration files into the project directory adds more flexibility because these files are reloaded each time the tool is run, which allows changing parameters 
+between runs without loading the project.
+
+As an example, consider changing the maximum number of frames to use in the feature tracker.  First copy **gui_track_features.conf** into your project directory.  Open this file and 
+look for the following section:
+
+.. code-block:: bash
+
+   # Parameters for the feature tracker
+   block feature_tracker
+     include core_feature_tracker.conf
+
+     # The maximum number of frames for the GUI to process.
+     # The tracker will choose frames distributed over the video
+     max_frames = 500
+   endblock
+
+
+Modify the line with “max_frame = 500” use a different value, such as 1000.  Note that you could also make this change in the project file by appending the following line:
+
+.. code-block:: bash
+
+   feature_tracker:max_frames = 1000
+
+Note that the "max_frames" parameter is in the "feature_tracker" scope and scope must be specified either using block/endblock notation or with a prefix before a colon.
+
+Printing all KLV metadata
+===========================
+
+The TeleSculptor application loads KLV metadata and display it in a viewer, but there is no way to export this data in batch.  However, the installer does provide the kwiver command 
+line tool that has an applet that will print out all metadata in a video.  This applet is called “dump_klv”. The default installation path is
+
+**C:\Program Files\TeleSculptor 1.1.0\bin\kwiver.exe**

```

Comment:
The `\` character is an escape character and so these paths don't format correctly.  I think you need to use one of the following:

    **C:\\Program Files\\TeleSculptor 1.1.0\\bin\\kwiver.exe**
    ``C:\Program Files\TeleSculptor 1.1.0\bin\kwiver.exe``

or in a literal block like this

    The default installation path is::

        C:\Program Files\TeleSculptor 1.1.0\bin\kwiver.exe

note the `::` and the indentation.

Reply:
I've been thinking that we might also want to replace all instances of the current TeleSculptor version with a substitution like this

    **C:\\Program Files\\TeleSculptor |version|\\bin\\kwiver.exe**

    **“C:\\Program Files\\TeleSculptor |version|\\bin\\kwiver.exe” dump_klv video_file.mpeg**

and then at the end of the file, or probably in a file that gets included, we have

    .. |version| replace:: 1.1.0 

This way when we update the version we only have one place to change it.  Or possibly in the future we can have CMake write this file with the correct version number in it.